app/test/[platform]/Content/index.tsx

```diff
@@ -86,7 +86,7 @@ export default function Section({
         const baseUrl = await platformApi({ render: true, action: 'download' }, platform, 'pads', true);
         const params = new URLSearchParams();
         idz.forEach(id => params.append('pads', id.toString()));
-        const url = `${baseUrl}?${params.toString()}`;
+        const url = `${baseUrl}&${params.toString()}`;
```

Comment:
Yes, by default the url return contains params like include_data and all that are available via the function. For the download link use case, this will always be true.